### PR TITLE
feat: add dashboard to (grafana) configmap(s)

### PR DESCRIPTION
**Pull request by psturc**
### Motivation
https://issues.jboss.org/browse/AEROGEAR-8512

### Verification steps
1. Spin up minishift and/or configure ASB on your cluster with `psturc` organisation and tag `AEROGEAR-8512`
1. Provision metrics-apb
1. Provision sync-app-apb with params:
    * docker image: `docker.io/psturc/voyager-server-example-metrics`
    * metrics endpoint: `metrics`
    * sync app port: `4000`
    * rest of params doesn't matter
1. Verify `sync-app-<your-app-name>-dashboard.json` is present in `grafana-dashboards-configmap`
1. Go to Grafana and expand list of dashboards - and Sync app dashboards should be present
1. Rest of the verification should be covered in https://github.com/aerogearcatalog/metrics-apb/pull/92

### CODE REVIEW

**Comment by aliok** [comment]
Code looks good. Gonna verify together with https://github.com/aerogearcatalog/metrics-apb/pull/92

File: roles/provision-sync-app-apb/tasks/main.yml [hunk 1]
```diff
@@ -159,4 +159,36 @@
...
+    configmap: "{{ dashboards_configmap }}"
+    namespace: "{{ namespace }}"
+
+- block:
+  - name: "Create {{ sync_dashboard_filename }} configmap if metrics-apb is not provisioned yet"
+    shell: oc create configmap '{{ sync_dashboard_filename }}' --from-file={{ sync_dashboard_filename }}={{ sync_dashboard_filepath }} -n '{{ namespace }}'
+  
+  - name: "Apply label 'sync-app-{{ SYNC_APP_NAME }}' to {{ sync_dashboard_filename }} configmap"
+    shell: oc label configmap {{ sync_dashboard_filename }} app=sync-app-{{ SYNC_APP_NAME }} -n {{ namespace }}
+  when: check_configmap.rc != 0 and encode_asb_binding
```
Comment (wei-lee) [inline roles/provision-sync-app-apb/tasks/main.yml]: I can't remember how this works now. But how does grafana pick this dashboard up?

Comment (aliok) [reply roles/provision-sync-app-apb/tasks/main.yml]: There's a related PR which takes the config map here and adds it to Grafana dashboard config map. https://github.com/aerogearcatalog/metrics-apb/pull/92

Comment (wei-lee) [reply roles/provision-sync-app-apb/tasks/main.yml]: excellent, thanks.

**Pull request merged**